backend/audit/models.py

```diff
@@ -475,12 +475,22 @@ class Meta:
         ]
 
 
+def excel_file_path(instance, filename):
+    """
+    We want the actual filename in the filesystem to be unique and determined
+    by report_id and form_section--not the user-provided filename.
+    """
+    base_path = "excel"
+    report_id = SingleAuditChecklist.objects.get(id=instance.sac.id).report_id
+    return f"{base_path}/{report_id}--{instance.form_section}.xlsx"
```

Comment:
Technically we [allow both `xls` and `xlsx`](https://github.com/GSA-TTS/FAC/blob/6b9cf8b34c558ba2209af6c6aa4635c0a1337d2a/backend/audit/validators.py#L31) extensions here, so we should probably extract and reuse the extension from the user-provided filename (at this point it will have already been validated to be either `xls` or `xlsx`)

Reply:
Alternatively, we should remove `xls` as an allowed extension. Actually starting to lean in this direction, considering the uploaded Excel files originate from us as `xlsx`

Reply:
I'm going to remove `.xls` from the list of allowed extensions.